icu4c/source/common/ubrk.cpp

```diff
@@ -174,6 +174,22 @@ ubrk_safeClone(
     return (UBreakIterator *)newBI;
 }
 
+U_CAPI UBreakIterator * U_EXPORT2
+ubrk_clone(const UBreakIterator *bi, UErrorCode *status) {
+    if (status == NULL || U_FAILURE(*status)) {
```

Comment:
I know that some existing code checks against `status == NULL` but we have a policy not to do that, so please don't in this new function.

Reply:
Done.

Reply:
not done yet

Reply:
um... which part is not done yet ?

Reply:
You just did this by accepting my edit :-)